Renderer.js

```diff
@@ -0,0 +1,54 @@
+
+class Renderer {
+
+    renderMainUser() {
+        $('.user-container').empty()
```

Comment:
You can chain the `empty()` before the `append()` and save a line of code, see the comment below: